CIScripts/Test/Tests/Integration/vRouterAgentService.Tests.ps1

```diff
@@ -12,56 +12,45 @@ Param (
 . $PSScriptRoot\..\..\..\Common\VMUtils.ps1
 . $PSScriptRoot\..\..\PesterHelpers\PesterHelpers.ps1
 
-. $ConfigFile
-$TestConf = Get-TestConfiguration
 $Sessions = New-RemoteSessions -VMs (Read-TestbedsConfig -Path $TestenvConfFile)
 $Session = $Sessions[0]
 
 $ControllerConfig = Read-ControllerConfig -Path $TestenvConfFile
+$OpenStackConfig = Read-OpenStackConfig -Path $TestenvConfFile
+$SystemConfig = Read-SystemConfig -Path $TestenvConfFile
 
 Describe "vRouter Agent service" {
-    Context "enabling" {
```

Comment:
Why is this removed? Maybe we should just mark this test as disabled?

Reply:
The test was identical to "runs correctly"